lms/lmsdb/bootstrap.py

```diff
@@ -237,6 +237,13 @@ def _api_keys_migration() -> bool:
     return True
 
 
+def _shareable_migration() -> bool:
+    Solution = models.Solution
```

Comment:
variable in camel case in a function? why like that?

Reply:
It's going to be deleted all the changes in this file (bootstrap), don't worry.
About the camel case - ask the one who wrote the api migration function - took the idea from there